lib/patches/bundler/ruby_version.rb

```diff
@@ -0,0 +1,29 @@
+module Bundler
+  class RubyVersion
+    def self.system
+      ruby_engine         = if defined?(RUBY_ENGINE) && !RUBY_ENGINE.nil?
+                              RUBY_ENGINE.dup
+                            else
+                              # not defined in ruby 1.8.7
+                              "ruby"
+                            end
```

Comment:
yeah, auto-formatted by mistake.

(I got used to them only because I prefer aligned hash's keys and values and there is only one option for both in idea and I won't go against auto-formatter.)